charts/hono/values.yaml

```diff
@@ -1718,15 +1718,31 @@ kafka:
     # Note that this could require creating RBAC rules, for more information refer to
     # https://github.com/bitnami/charts/tree/master/bitnami/kafka#accessing-kafka-brokers-from-outside-the-cluster
     create: true
+  # limit Kafka JVM's memory consumption
+  heapOpts: "-Xms620M -Xmx620M"
+  resources:
+    requests:
+      memory: "800Mi"
+    limits:
+      memory: "800Mi"
+
   zookeeper:
     persistence:
       enabled: false
     auth:
-      enabled: true
-      clientUser: "zookeeperUser"
-      clientPassword: "zookeeperPassword"
-      serverUsers: "zookeeperUser"
-      serverPasswords: "zookeeperPassword"
+      client:
+        enabled: true
+        clientUser: "zookeeperUser"
+        clientPassword: "zookeeperPassword"
+        serverUsers: "zookeeperUser"
+        serverPasswords: "zookeeperPassword"
+    # limit Zookeeper JVM's memory consumption
+    heapSize: 400
+    resources:
+      requests:
+        memory: "500Mi"
```

Comment:
How about keeping the zookeeper default of `256Mi` here for the `requests` value? I would also consider reducing the corresponding kafka resources/requests/memory configuration. Both would reduce the overall minimal memory requirements which could for example be relevant when installing the cloud2edge chart locally for testing purposes.

Reply:
I had replaced this PR with #438 ...

Reply:
Sorry, mixed up the 2 PRs.